tests/system/feature_store/test_feature_store.py

```diff
@@ -2489,6 +2488,61 @@ def test_get_offline_for_two_feature_set_with_same_column_name(self):
         for key in res.to_dataframe().to_dict().keys():
             assert key in expected
 
+    @pytest.mark.parametrize(
```

Comment:
I don't see a reason for this test to be a system test (expensive/slow) and not a unit test (cheap/fast)